legend-engine-xt-serviceStore-pure/src/main/resources/core_servicestore/executionPlan/executionPlan_generation.pure

```diff
@@ -48,6 +48,7 @@ import meta::external::store::service::metamodel::mapping::*;
 import meta::external::store::service::functions::pureToServiceStoreQuery::*;
 import meta::external::store::service::router::systemMapping::*;
 import meta::external::store::service::utils::helperFunction::*;
+import meta::pure::functions::collection::*;
```

Comment:
lets move this import near meta::pure imports

Reply:
done